src/models/user.js

```diff
@@ -0,0 +1,21 @@
+import { DataTypes } from 'sequelize';
+import { client } from '../utils/db.js';
+
+export const User = client.define('user', {
+  name: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  email: {
+    type: DataTypes.STRING,
+    allowNull: false,
+    unique: true,
+  },
+  password: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  activationToken: {
+    type: DataTypes.STRING,
```

Comment:
Consider adding a default value or setting allowNull to true for 'activationToken' if it's optional. Otherwise, it might cause issues when creating a user without this field.